cinema/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from cinema.views import movie_list, movie_details
+
+urlpatterns = [
+    path("movies/", movie_list, name="movie_list"),
+    path("movies/<int:pk>", movie_details, name="movie_details"),
```

Comment:
Ensure all your endpoints end with a `/` to maintain consistency and avoid potential issues with URL matching. Consider changing `path("movies/<int:pk>", movie_details, name="movie_details")` to `path("movies/<int:pk>/", movie_details, name="movie_details")`.